rfcs/text/0008-threat-intel.md

```diff
@@ -87,11 +84,11 @@ Stage 3: Add or update all remaining field definitions. The list should now be e
 Stage 1: Describe at a high-level how these field changes will be used in practice. Real world examples are encouraged. The goal here is to understand how people would leverage these fields to gain insights or solve problems. ~1-3 paragraphs.
 -->
 
-The additions described above will be used to enable cyber threat intelligence capabilities in Elastic Security solution. A new rule type Indicator match will be introduced in 7.10 and the proposed ECS updates will enable a new category of detection alerts that match incoming log and event data against threat intelligence sources. Additionally in the future we will also develop enrichment flows that add context from threat intelligence to alerts and events to assist analysts in their investigative workflows.
+The additions described above will be used to enable cyber threat intelligence capabilities in Elastic Security solution. A new rule type Indicator match was introduced in 7.10 and the proposed ECS updates will enable a new category of detection alerts that match incoming log and event data against threat intelligence sources. Additionally in the future we will also develop enrichment flows that add context from threat intelligence to alerts and events to assist analysts in their investigative workflows.
 
-There are two primary uses for these fields.
+While there are two primary uses for these fields, this RFC deals primarily with the first: ingestion/storage of threat intelligence.
 
-1. **Storing threat intelligence as an event document in threat index(s).**
+**Storing threat intelligence as an event document in threat index(s).**
 
     Threat intelligence data will be collected from multiple sources stored in threat indices. The ECS fields proposed here will be used to structure the documents collected from various sources.
 
```

Comment:
@peasead the example below declares `indicator.*` instead of `threat.indicator.*`, and as mentioned by @ebeahan is also missing many of the fields in the above table.

Reply:
Yep. I'm working that.

Reply:
yessir. I got that buttoned up.